BR: Add a new doc about the batch create table (#7983)
pingcap/docs: br/backup-and-restore-faq.md

Yes. BR backs up the [`SHARD_ROW_ID_BITS` and `PRE_SPLIT_REGIONS`](/sql-statements/sql-statement-split-region.md#pre_split_regions) information of a table. The data of the restored table is also split into multiple Regions.

> ## What should I do if the `entry too large, the max entry size is 6291456, the size of data is 7690800` error is reported when restoring data using BR?

Reviewer: What should I do if the restore failure with error message the `entry too large, the max entry size is 6291456, the size of data is 7690800` using BR?

Reviewer: it looks 'using BR' can removed also, since this chapter is all about BR FAQ

> ## What should I do if the `entry too large, the max entry size is 6291456, the size of data is 7690800` error is reported when restoring data using BR?

Author: ```suggestion
## What should I do if the restore fails with the error message `the entry too large, the max entry size is 6291456, the size of data is 7690800`?
```

> ## What should I do when the `entry too large, the max entry size is 6291456, the size of data is 7690800` error reported during data restoration using BR?

Author: ```suggestion
## What should I do when the `entry too large, the max entry size is 6291456, the size of data is 7690800` error is reported during data restoration using BR?
```

Try reducing the size of the concurrent batch table creation by setting `--ddl-batch-size` to `128` or a smaller value.

> Suppose that the value of [`--ddl-batch-size`](/br/br-batch-create-table.md#how to use) is greater than `1`, and you use BR to restore the backup data. In this case, TiDB writes the queue of DDL jobs that create tables to TiKV. At this time, the total schema size of all tables sent by TiDB at a time should not exceed 6 MB, because the default maximum value of job messages that TiDB can send at a time is `6 MB` (you are **not recommended** to modify this value; for details, see [`txn-entry-size-limit`](/tidb-configuration-file.md#txn-entry-size-limit-new-in-v50) and [`raft-entry-max-size`](/tikv-configuration-file.md#raft-entry-max-size). Therefore, if you set the too large value to `--ddl-batch-size`, the schema size of the batch table sent by TiDB at a time exceeds the specified value, resulting in BR reporting `entry too large, the max entry size is 6291456, the size of data is 7690800` error.

Reviewer: ```suggestion
When using BR to restore the backup data with the value of [`--ddl-batch-size`](/br/br-batch-create-table.md#how to use) is greater than `1`, and you . TiDB writes a DDL job to the DDL jobs queue that maintained by TiKV. At this time, the total size of all tables schema sent by TiDB at a time should not exceed 6 MB, because the maximum value of job messages is `6 MB` by default (you are **not recommended** to modify this value; for details, see [`txn-entry-size-limit`](/tidb-configuration-file.md#txn-entry-size-limit-new-in-v50) and [`raft-entry-max-size`](/tikv-configuration-file.md#raft-entry-max-size). Therefore, if you set the too large value to `--ddl-batch-size`, the schema size of the batch table sent by TiDB at a time exceeds the specified value, resulting in BR reporting `entry too large, the max entry size is 6291456, the size of data is 7690800` error.
```
